core/src/main/kotlin/transformers/pages/PageNodeTransformer.kt

```diff
@@ -4,6 +4,6 @@ import org.jetbrains.dokka.pages.ModulePageNode
 import org.jetbrains.dokka.pages.RootPageNode
 import org.jetbrains.dokka.plugability.DokkaContext
 
-interface PageNodeTransformer {
-    operator fun invoke(input: RootPageNode): RootPageNode
+abstract class PageNodeTransformer(context: DokkaContext? = null) {
+    abstract operator fun invoke(input: RootPageNode): RootPageNode
 }
```

Comment:
This change is probably unnecessary

Reply:
Yup. Extension points' interfaces(!) shouldn't be aware of context. Only their implementation should. It is done this way in every other EP.